lib/aws_auditor/scripts/audit.rb

```diff
@@ -19,32 +20,56 @@ def self.execute(environment, options=nil)
           tag_name = options[:tag]
         end
 
+        slack = options[:slack]
+        puts "Condensed results from this audit will print into Slack instead of directly to an output." if slack
+
         no_selection = !(options[:ec2] || options[:rds] || options[:cache])
 
-        output("EC2Instance", tag_name) if options[:ec2] || no_selection
-        output("RDSInstance", tag_name) if options[:rds] || no_selection
-        output("CacheInstance", tag_name) if options[:cache] || no_selection
+        output("EC2Instance", tag_name, slack, environment) if options[:ec2] || no_selection
+        output("RDSInstance", tag_name, slack, environment) if options[:rds] || no_selection
+        output("CacheInstance", tag_name, slack, environment) if options[:cache] || no_selection
       end
 
-      def self.output(class_type, tag_name)
+      def self.output(class_type, tag_name, slack, environment)
```

Comment:
This method is getting a bit messy and doing too much.  I think we should refactor it a bit to separate the gathering of data and the presentation.  With those two concerns separated, the terminal vs slack output should be easy and succinct.